Modules/Log/Get-RegistryKeyValData.ps1

```diff
@@ -0,0 +1,169 @@
+<#
+.SYNOPSIS
+Get-RegistryKeyValData.ps1 retrieves the value of the provided key as
+well as the last modified time of the key.
+
+.NOTES
+Next line needed by Kansa.ps1 for proper handling of this script's data
+OUTPUT tsv
+
+HKEY_LOCAL_MACHINE
+HKEY_USERS
+HKEY_CURRENT_USER is a subkey of HKEY_USERS
+HKEY_CURRENT_CONFIG is a subkey (HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Hardware Profiles\Current)
+HKEY_CLASSES_ROOT is a subkey (HKEY_LOCAL_MACHINE\SOFTWARE\Classes)
+#>
+param(
+[string]$Key
+)
+
+$Error.Clear()
+$ErrorActionPreference = "SilentlyContinue"
+
+## The following code is from
+## Name: Get-RegistryKeyTimestamp
```

Comment:
What license was this script released under? Attribution is great, but we need to make sure it's compatible as well.